Assets/Scripts/LevelGeneration/MapGenerator.cs

```diff
@@ -18,7 +18,9 @@ public class MapGenerator : MonoBehaviour
     public Transform generatorPoint;
 
     public float xOffset = 18f, yOffset = 10f;
-
+    private float xWallSize, yWallSize;
```

Comment:
can you prefix the private vars with _ ex. _xWallSize, _yWallSize, just a quick style thing